tests/cases/conformance/jsdoc/jsdocDeprecatedTag1.ts

```diff
@@ -0,0 +1,37 @@
+// @declaration: true

```

Comment:
Unfortunately, without suggestions getting logged as errors, these tests aren't worthwhile. I guess fourslash is the only way we have to test suggestion messages and spans right now.